tests/tests_example_scene.py

```diff
@@ -0,0 +1,52 @@
+from manimlib.imports import *
+
```

Comment:
Shouldn't this file be named `testing_sample_scenes.py` or something to be consistent with the other sample file?

Reply:
Yup. Changed